usaspending_api/disaster/v2/views/overview.py

```diff
@@ -50,40 +71,47 @@ def funding(self):
             for elem in raw_values
         ]
 
-    def spending(self, funding):
+    def spending(self, funding, defc):
         return {
-            "award_obligations": self.award_obligations(),
-            "award_outlays": self.award_outlays(),
-            "total_obligations": self.total_obligations(funding),
-            "total_outlays": self.total_outlays(),
+            "award_obligations": self.award_obligations(defc),
+            "award_outlays": self.award_outlays(defc),
+            "total_obligations": self.total_obligations(funding, defc),
+            "total_outlays": self.total_outlays(defc),
         }
 
-    def award_obligations(self):
+    def award_obligations(self, defc):
         return (
-            FinancialAccountsByAwards.objects.filter(disaster_emergency_fund__in=covid_def_code_strings())
```

Comment:
I think this was the only place `covid_def_code_strings()` was used. Should `covid_def_code_strings()` and `covid_def_codes()` be removed?

Reply:
They should, and now they have been.